code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -297,9 +297,19 @@
 	name = "elite autoinjector"
 	desc = "An elite auto-injector loaded with a strong and versatile combination of chemicals, healing most types of damage. Issued almost excusively to the infamous Nanotrasen deathsquads"
 	icon_state = "autoinjector-7"
+	volume = 60
+	amount_per_transfer_from_this = 30
 	list_reagents = list(
```

Comment:
Errr what the hell? Tea? Can you just keep it to rezadone and RR please?

Reply:
it's so joever...